website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -119,6 +129,30 @@ dbt sl export --saved-query sq_number1 --export-as table --alias new_export
 ```
 </details>
 
+### Exports for multiple saved queries
+
+Use the command, `dbt sl export-all`, to run exports for multiple saved queries at once. This is different to the `dbt sl export` command, which only runs exports for a singular saved query.  For example, to run exports for multiple saved queries, you can use:
+
+#### Example
```

Comment:
I think we can just go right into it without the header since you set the expectation in the previous sentence
```suggestion
```